docs/privacy-policy.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: doc
+title: Privacy Policy
+permalink: /docs/privacy/
+hide: true
+---
+
+This privacy policy will explain how Hackathons for Schools Association (“us”, “we”, “our”) uses the personal data we collect from you when you use hackathonsforschools.com (“our websites”) and our Discord server or Google Forms (“our services”).
+
+Topics:
+
+- What data do we collect?
+- How do we collect your data?
+- How will we use your data?
+- How do we store your data?
+- Marketing
+- What are your data protection rights?
+- What are cookies?
+- What types of cookies do we use?
+- How to manage your cookies
+- Privacy policies to other websites
+- Changes to our privacy policy
+- How to contact us
+- How to contact the appropriate authorities
+- What data do we collect?
+- We collect the following data:
+
+## What data do we collect?
+
+- Personal identification information (name, email address, phone number, - social media usernames)
+
+## How do we collect your data?
+
+You directly provide us with most of the data we collect. We collect data when you:
+
+- When you register online on any of our websites or services
+- Voluntarily fill out public profile information on any of our websites
+- Submit a workshop, talk or other similar type of application
+- Create posts on any of our websites or services
+- Use or view our website via your browser’s cookies
+
+## How we will use your data?
+
+We collect your data so that we can:
+
+- Ensure the adequate safeguarding of minors engaging in our activities
+- Email you about relevant information pertaining to your engagement with - our activities
+- To review project submissions
+- To review talk and workshop applications
+- To schedule our activities
+- To enforce the [Hackathons for Schools Association Code of Conduct](/code-of-conduct)
+
+## How we store your data?
+
+Data collected via Google Drive services, including Google Forms, are stored on William Russell's Google Drive account. This is only shared between the Hackathons for Schools Association Core Team.
+
+Due to a change in [Google's Terms and Service](https://policies.google.com/terms/update?hl=en-US), this data may be stored in either Europe or the United States of America.
+
+We may keep any data we collect about you for up to 12 months. Once this time period has expired we will delete your data.
+
+## Marketing
```

Comment:
I'm not sure this should really be a possibility. Marketing to kids just feels wrong to me?

Reply:
Yeah agreed. Can you remove it please